File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/features/FeatureHighlightItem.java
package com.philips.cdp.prxclient.datamodels.features;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class FeatureHighlightItem implements Serializable {

	private static final long serialVersionUID = 4150127431517255657L;
	@SerializedName("featureCode")
	public String featureCode;

	@SerializedName("featureHighlightRank")
	public String featureHighlightRank;

	@SerializedName("featureReferenceName")
	public String featureReferenceName;
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/features/FeatureHighlightItem.kt
package com.philips.cdp.prxclient.datamodels.features

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class FeatureHighlightItem(
        var featureCode: String? = null, var featureHighlightRank: String? = null, var featureReferenceName: String? = null) : Parcelable